src/modules/CSADigitizer/README.md

```diff
@@ -82,5 +89,18 @@ clock_bin_tot = 25.0ns
 
 
 
+```ini
```

Comment:
Maybe add a comment above what this one is for (i.e. MuPix10)? And remove the excessive newlines after.